File: API/Helpers/MemberParams.cs
namespace API.Helpers
{
    public enum eMemberType
    {
        Scout = 0,
        Adult = 1,
        AgeScout = 2
    };
    public class MemberParams : PaginationParams
    {
        public string LastName { get; set; }
        public string FirstName { get; set; }
        public int PatrolId { get; set; }
        public string OrderBy { get; set; } = "Name";
        public eMemberType MemberType { get; set; } 
    }
}

File: client/src/app/_models/memberParams.ts
import { MemberType } from './../_enum/memberTypeEnum';

export class MemberParams { 
    lastName: string = "";
    firstName: string = "";
    patrolId: string = "0";
    pageNumber: number = 1;
    pageSize: number = 12;
    orderBy: string = "name";
    memberType: MemberType = 0;
}